File: SportBackend/src/main/java/com/dlust/sportbackend/entity/TeamScoreAdjustment.java
package com.dlust.sportbackend.entity;

import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
public class TeamScoreAdjustment {
    private Long id;
    private Long teamId;
    private BigDecimal deltaAmount;
    private String note;
    private LocalDateTime createTime;
}

File: webManageProject/src/api/teamScoreAdjustment.ts
import request from '@/utils/request'

export interface TeamScoreAdjustment {
  id: number
  teamId: number
  deltaAmount: number
  note: string
  createTime: string
}

export function getAdjustmentList(teamId: number) {
  return request.get('/admin/teamScoreAdjustment/list', { params: { teamId } })
}

export function addAdjustment(data: { teamId: number; deltaAmount: number; note: string }) {
  return request.post('/admin/teamScoreAdjustment/add', data)
}

export function deleteAdjustment(id: number) {
  return request.delete(`/admin/teamScoreAdjustment/${id}`)
}
